includes/definitions/discovery/audiocodes.yaml

```diff
@@ -0,0 +1,153 @@
+mib: AC-ANALOG-MIB:AcGateway:AC-PSTN-MIB:DS1-MIB:AC-PM-PSTN-MIB
+modules:
+    sensors:
+        pre-cache:
+            data:
+                - oid:
+                    - callerDisplayCallerDisplay
+                    - channelsStartingPhoneNum
+                    - acAnalogFxsFxoType
+                    - acTrunkDialPlanName
+                    - dsx1CircuitIdentifier
+                    - acPMTrunkUtilizationAttributes
+        state:
+            data:
+                -
+                    oid: acAnalogFxsFxoTable
+                    value: acAnalogFxsFxoHookState
+                    num_oid: .1.3.6.1.4.1.5003.9.10.1.4.1.1.18.{{ $index }}
+                    descr: 'FXO {{ $index }} ({{ $callerDisplayCallerDisplay }}, {{ $channelsStartingPhoneNum }})'
+                    index: 'hookStatus{{ $index }}'
+                    state_name: acAnalogFxsFxoHookState
+                    states:
+                        - { descr: offHook, graph: 0, value: 0, generic: 0 }
+                        - { descr: onHook, graph: 0, value: 1, generic: 0 }
+                    skip_values:
+                        -
+                            oid: acAnalogFxsFxoType
+                            op: '!='
+                            value: 0
+                -
+                    oid: acAnalogFxsFxoTable
+                    value: acAnalogFxsFxoHookState
+                    num_oid: .1.3.6.1.4.1.5003.9.10.1.4.1.1.18.{{ $index }}
+                    descr: 'FXS {{ $index }} ({{ $callerDisplayCallerDisplay }}, {{ $channelsStartingPhoneNum }})'
+                    index: 'hookStatus{{ $index }}'
+                    state_name: acAnalogFxsFxoHookState
+                    states:
+                        - { descr: offHook, graph: 0, value: 0, generic: 0 }
+                        - { descr: onHook, graph: 0, value: 1, generic: 0 }
+                    skip_values:
+                        -
+                            oid: acAnalogFxsFxoType
+                            op: '!='
+                            value: 1
+                -
+                    oid: acAnalogFxsFxoTable
+                    value: acAnalogFxsFxoHookState
+                    num_oid: .1.3.6.1.4.1.5003.9.10.1.4.1.1.18.{{ $index }}
+                    descr: 'E&M {{ $index }} ({{ $callerDisplayCallerDisplay }}, {{ $channelsStartingPhoneNum }})'
+                    index: 'hookStatus{{ $index }}'
+                    state_name: acAnalogFxsFxoHookState
+                    states:
+                        - { descr: offHook, graph: 0, value: 0, generic: 0 }
+                        - { descr: onHook, graph: 0, value: 1, generic: 0 }
+                    skip_values:
+                        -
+                            oid: acAnalogFxsFxoType
+                            op: '!='
```

Comment:
should this be `=`?  I assume 2 is not present/configured.

Reply:
The idea is to create 3 times the same OID depending on the value of acAnalogFxsFxoType but change the description statically in each of the 3 instance. Unfortunately, there is no way to get "FXO", "FXS" of "E&M" from a variable ... So the '!=' is correct here.